README.md

```diff
@@ -233,7 +234,56 @@ Check the  [Notification Configuration](#37-notification-configuration) to see h
 For more information, please check the [Global Setting Configuration](#38-global-setting-configuration)
 
 
-### 1.4 Administration
+### 1.4 Channel
+
+The Channel is used for connecting the Probers and the Notifiers. It can be configured for every Prober and Notifier.
+
+This feature could help you group the Probers and Notifiers into a logical group.
+
+> **Note**:
+> 
+> 1) If you don't define the Channel, the default channel will be used for these probers and notifiers. The default channel name is `__EaseProbe_Channel__` 
+> 
+> 2) Versions of EaseProbe prior to  v1.5.0, do not support the `channel` feature
+
+```YAML
+
+For example:
+
+```YAML
+http: 
+   - name: probe A
+     channels : [ Dev_Channel, Manager_Channel ]
```

Comment:
Do the channel names need to follow any particular rules? 
Such as no space, no symbols etc?

Reply:
looks like the limitation is just only for yaml file.  If the chars can be accepted by yaml, then we can support that.